jupyterfs/fsmanager.py

```diff
@@ -204,9 +205,9 @@ def _base_model(self, path):
         try:
             model['writable'] = info.permissions.check('u_w')  # TODO check
         except (errors.MissingInfoNamespace,):
-            # if relevant namespace is missing, assume writable
-            # TODO: decide if this is wise
-            model['writable'] = True
+            # if relevant namespace is missing,
+            # fallback to default access
```

Comment:
```suggestion
            # if relevant namespace is missing, use default
```

try to keep code comments to one line